### ISSUE TITLE
Miniseq LearnErros

### ISSUE THREAD

**Issue by RFlazcano** [benjjneb/dada2#1486]
I am working with 16S data sequenced on a Miniseq instrument and I am having difficulty interpreting these error plots where the fitted model drops off at higher quality scores. It seems others have had similar problems with data where quality scores are binned. The learerrors command runs to completion and I am using  nbases = 1E10.  Most of the points above the fitted line are of low frequency so I am wondering if it would be ok to proceed with this error model and how to interpret the plots. 

[F_err.pdf](https://github.com/benjjneb/dada2/files/7961511/F_err.pdf)
[F_err_freq.pdf](https://github.com/benjjneb/dada2/files/7961512/F_err_freq.pdf)
[R_err.pdf](https://github.com/benjjneb/dada2/files/7961513/R_err.pdf)
[R_err_freq.pdf](https://github.com/benjjneb/dada2/files/7961515/R_err_freq.pdf)

**Comment by benjjneb** [comment]
You're OK to proceed. It looks ugly on binned data, but still seems to work reasonably well. You can read much more at the main thread for DADA2 and binned quality scores: #1307

**Comment by vebaev** [comment]
@RFlazcano sorry to interfere here (i don't have ways to contact you other ways), but I was intrigued as I don't see info in the internet using 16s on miniSeq? How was your experience? i think miniSeq do not provide 250 PE which is like a standard for 16s nowadays (if you want to do v3v4 or v1v3)? How much samples you can multiplex if you want to get like 100k per sample?

**Comment by RFlazcano** [comment]
@vebaev. I did not find much on 16S and MiniSeq besides a couple papers such as this one: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6291791/

We used the Miniseq because the sequencing core facility my University uses recommended it because it would be less expensive and the quality is better because it is 2x150 and we were only sequencing V4.  The only issue I ran into was with interpreting the error model plots because of the binned quality scores.  Otherwise it was pretty straightforward.  I had no issues with merging either because the reads were pretty good quality coming off the sequencer so there was not really a drop off in quality at the ends of reads as is standard with the MiSeq so I did not trim the end. In regards to multiplexing to get ~100k reads per sample I am not sure since our sequencing core facility handles all of that.
